install-daemon.sh

```diff
@@ -155,6 +161,50 @@ function check_os_comp {
     print_error "Unsupported OS"
     exit 1
   fi
+  if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ] || [ "$OS" == "zorin" ]; then
+    apt-get update
+
+    # install virt-what
+    apt-get install -y virt-what
```

Comment:
While I do not oppose the use of `virt-what`, are you aware of any other method of detecting virtualization that do not require the extra dependency? (maybe in `/proc/cpuinfo`?)

Reply:
I thought about that, but when trying out the commands like `lscpu`, `sudo dmidecode -s system-manufacturer` I didn't get persistent results on my vm.

Reply:
Hmm, alright. I'll let it go for now.